src/collector/top_score_collector.rs

```diff
@@ -850,6 +850,21 @@ where
         self.buffer
     }
 
+    /// Returns the elements between `offset` and `top_n` in sorted order.
+    pub fn into_sorted_after(mut self, offset: usize) -> impl Iterator<Item=ComparableDoc<Score, D, R>> {
+        if self.buffer.len() > self.top_n {
+            self.truncate_top_n();
+        }
+
+        if offset >= self.buffer.len() {
+            return vec![].into_iter().skip(0);
+        }
+
+        let (_, _, remainder) = self.buffer.select_nth_unstable(offset);
+        remainder.sort_unstable();
+        self.buffer.into_iter().skip(offset)
```

Comment:
I am not convinced this is actually beneficial, do you have any benchmarks demonstrating that doing these two effective sorts is more efficient than the one sort?

Reply:
`select_nth` is not a sort: it only pivots elements around the `nth` position (in linear time).

Reply:
No, I would create a set of micro benchmarks between the two changes.

Reply:
I went ahead and dropped this change, since the benchmarks weren't showing a meaningful improvement for relevant window sizes. Thanks!